{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/settings.py

```diff
@@ -241,7 +241,7 @@
 if USE_S3:
     AWS_S3_CUSTOM_DOMAIN = env.str("AWS_S3_CUSTOM_DOMAIN", "")
     AWS_S3_OBJECT_PARAMETERS = {"CacheControl": "max-age=86400"}
-    AWS_DEFAULT_ACL = env.str("AWS_DEFAULT_ACL", "public-read")
+    AWS_DEFAULT_ACL = env.str("AWS_DEFAULT_ACL", "bucket-owner-full-control")
```

Comment:
What happened to public read?

Reply:
By default, S3 buckets are created with ACL disabled and object ownership is enforced to the Bucket owner. That's why when we use `bucker-owner-full-control` it works.

Reply:
Okay, as long as it works and public can view it with that signed url, then it should be fine

Reply:
We've tested with @ishubham1995 and it's working.